src/usr/local/www/includes/functions.inc.php

```diff
@@ -33,12 +33,13 @@ function get_stats() {
 	$stats['states'] = get_pfstate();
 	$stats['temp'] = get_temp();
 	$stats['datetime'] = update_date_time();
-	$stats['interfacestatistics'] = get_interfacestats();
-	$stats['interfacestatus'] = get_interfacestatus();
+	$stats['interfacestatistics'] = 0;//get_interfacestats();//unused
+	$stats['interfacestatus'] = 0;//get_interfacestatus();//unused
 	$stats['cpufreq'] = get_cpufreq();
 	$stats['load_average'] = get_load_average();
-	$stats['mbuf'] = get_mbuf();
-	$stats['mbufpercent'] = get_mbuf(true);
+	get_mbuf($mbuf, $mbufpercent);
```

Comment:
Was thinking it would be slightly more 'readable' but changed it now, i'm okay with it either way.